src/main/kotlin/edu/cornell/cs/apl/viaduct/protocols/CommitmentProtocol.kt

```diff
@@ -5,19 +5,20 @@ import edu.cornell.cs.apl.viaduct.security.Label
 import edu.cornell.cs.apl.viaduct.syntax.Host
 import edu.cornell.cs.apl.viaduct.syntax.HostTrustConfiguration
 import edu.cornell.cs.apl.viaduct.syntax.Protocol
+import edu.cornell.cs.apl.viaduct.util.asComparable
 
-class CommitmentProtocol(val sender: Host, val recievers: Set<Host>) : Protocol {
+class CommitmentProtocol(val sender: Host, val receivers: Set<Host>) : Protocol {
```

Comment:
This is about Josh's code, but I just realized this: don't use `Set` in class fields. Always use `PersistentSet`. All other protocols take a `Set` in their constructor, but store a `PersistentSet`.

Reply:
fix this in another merge, this one is gigantic already